docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -597,3 +597,23 @@ spec:
   resolution: "Enroll device to MDM"
   platforms: macOS
   contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Application is up to date or not present (macOS) (Docker example)
+  query: SELECT 1 WHERE EXISTS (SELECT 1 FROM apps a1 WHERE a1.bundle_identifier = 'com.electron.dockerdesktop' AND a1.bundle_short_version>='4.6.1') OR NOT EXISTS (SELECT 1 FROM apps a2 WHERE a2.bundle_identifier = 'com.electron.dockerdesktop');
+  description: "Checks if Docker is installed and at version 4.6.1 or above, or not installed. Fails if Docker is installed and on a lower version."
+  resolution: "Update Docker or remove it if not used."
+  platforms: macOS
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: No unencrypted SSH keys are present
```

```suggestion
  name: SSH keys encrypted
```
What do you think about this name instead? I'm trying to match the current "<feature or item(s)> <desired state>" format that we're using for other policies like "Full disk encryption enabled."

---

In this case, if there are no SSH keys, it passes, so "SSH Keys encrypted" is not 100% accurate. There might be better wording though.

SSH keys not present or encrypted?

---

@GuillaumeRoss aha, thank you for the clarification.

I'm thinking, as a user, I care most that a device has encrypted SSH keys. I want to know which devices don't have have encrypted SSH keys.

I care less about whether or not the device has any SSH keys at all. I could be wrong here.

What if we used the "SSH keys encrypted" title and then relied on your sentence below to clarify that devices with no SSH keys detected are marked as passing?

`Policy passes if no unencrypted SSH keys are found in the ~/.ssh of users on the system.`

Could we simplify this sentence to...`If no SSH keys are found, a host is marked as passing.` ?